src/node/node-api.service.ts

```diff
@@ -41,15 +41,6 @@ export class NodeApiService {
     return await this.request.get(`${url}/transactions/info/${id}`);
   }
 
-  async sendTransaction(data: any): Promise<AxiosResponse | Error> {
```

Comment:
It's nice to keep the `sendTransaction` method, which will just do a sign and broadcast.